src/fileutils.hpp

```diff
@@ -40,7 +40,7 @@
 #endif
 
 std::string get_home_dir(void);
-void make_dir(std::string path, int perms=775);
+void make_dir(std::string path, int perms=0775);
```

Comment:
Whoa, good catch. Octal ftw.

Reply:
@jaycedowell Interesting note about the octal! You need `0775` for the filesystem side of the ifdef, but `775` was correct for the else branch... because it gets converted to a string as an arg for mkdir. Now that I'm testing both sides of these particular ifdefs, I uncovered the error `mkdir: invalid mode ‘509’` (the result of `to_string(0775)`, heh). Fixing.

Reply:
Things were easier when I ignored how this worked with pre-C++17 compatibility.